Part one answer

sky192 opened the pull request.

Kayla-SA-W commented:
@sky192  Your `great-novels.sql` file is missing information. I cannot grade this. Please resubmit your file with the correct sql workbench file.

sky192 commented:
> @sky192 Your `great-novels.sql` file is missing information. I cannot grade this. Please resubmit your file with the correct sql workbench file.

Hey Kayla, for some reasons I could not push my part-one-answer branch, It was rejected so I made another branch named part-1-answer and modified my files and fixed errors then push it, please review part-1-answer branch
